tests/contrib/cassandra/test.py

```diff
@@ -264,9 +264,15 @@ def test_batch_statement(self):
         spans = writer.pop()
         eq_(len(spans), 1)
         s = spans[0]
-        eq_(s.resource, 'BatchStatement')
+        eq_(
+            s.resource,
+            (
+                'INSERT INTO test.person_write (name, age, description) VALUES (\'Joe\', 1, \'a\'); '
+                'INSERT INTO test.person_write (name, age, description) VALUES (\'Jane\', 2, \'b\')'
+            )
+        )
         eq_(s.get_metric('cassandra.batch_size'), 2)
-        assert 'test.person' in s.get_tag('cassandra.query')
+        assert 'test.person' in s.resource
```

Comment:
we should probably be asserting that this shows up twice in the resource?

Reply:
or, we should know what the exact resource is, no? or does it change between versions?

Reply:
This can actually be removed since we test the resource exactly on line 267.